### DISCUSSION TITLE
Unable to call the carousel

### DISCUSSION THREAD

**Discussion by saleemhaq** [OrchardCMS/OrchardCore#10593] [Q&A]
I unable to call the helper method facing issue please see an attachment for understanding.

![injectIorchardHelper](https://user-images.githubusercontent.com/5202291/139508795-43f5ffe9-8ea1-4545-ad50-3e901642b14c.PNG)
![callingonLayout](https://user-images.githubusercontent.com/5202291/139508963-3d3e918d-159d-496e-a412-2a919be079b2.png)

**Comment by Skrypt** [comment]
You need to add a reference to OrchardCore.Alias module in the project where you are using this helper method.
See documentation here : https://docs.orchardcore.net/en/latest/docs/reference/core/Razor/
There is an indication on the modules to reference right next to the method.

**Reply by Skrypt** [reply]
See @ns8482e this is why https://github.com/OrchardCMS/OrchardCore/issues/10513

**Reply by saleemhaq** [reply]
> You need to add a reference to OrchardCore.Alias module in the project where you are using this helper method. See documentation here : https://docs.orchardcore.net/en/latest/docs/reference/core/Razor/ There is an indication on the modules to reference right next to the method.

![unabletouse](https://user-images.githubusercontent.com/5202291/139528969-8c0ad88f-689b-421a-b0ca-c195956e8331.png)
Still facing the issue: PFA

**Reply by jtkech** [reply]
Don't inject `IOrchardHelper`, just use `Orchard` which is a property of our custom `RazorPage` views

**Reply by saleemhaq** [reply]
> Don't inject `IOrchardHelper`, just use `Orchard` which is a property of our custom `RazorPage` views

I remove then i face this error pf
![facingissue](https://user-images.githubusercontent.com/5202291/139538685-1fd29c16-31ae-4f2a-9313-d766d1093ac5.png)
a.

**Reply by Skrypt** [reply]
You need to reference OrchardCore.Media project for that one.

**Reply by saleemhaq** [reply]
> You need to reference OrchardCore.Media project for that one.

I have a slider like that have numbers of banner see html below:

<div class="container h-100">
                        <div class="row h-100 align-items-center">
                            <div class="col-lg-6 col-md-8 col-sm-8 col-6">
                                <div class="banner-cont rounded-lg shadow p-3 p-md-4 p-xl-5">
                                    <figure><img src="/TheTheme/images/58761.png" alt="" class="img-product-logo mb-4"></figure>
                                    <ul class="list-style mb-4">
                                        <li>lorem epsum.</li>
                                        <li>lorem epsum.</li>
                                        <li>lorem epsum.</li>
                                        <li>lorem epsum.</li>
                                    </ul>
                                    <a href="#" class="btn btn-orange">Know More</a>
                                </div>
                            </div>
                        </div>
                    </div>

How i achieve with carousel? 

![slider1](https://user-images.githubusercontent.com/5202291/139554737-724bb989-6dbf-48cf-8dca-6b9ad58e4515.PNG)

**Reply by saleemhaq** [reply]
Anyone help me out?
